src/pages/lesson-details/LessonsDetails.styles.ts

```diff
@@ -19,6 +19,22 @@ export const styles = {
     display: 'flex',
     justifyContent: 'space-between'
   },
+  button: {
+    background: palette.basic.grey,
+    '&:hover': {
+      backgroundColor: 'primary.100'
+    },
+    color: palette.basic.black,
+    maxWidth: '97px'
```

Comment:
Put it in the order. Move hover at the end it will be more readable.

And you can use just 'basic.black' or 'basic.grey' without any import palette

Reply:
done